src/Microsoft.DocAsCode.Build.RestApi/RestApiDocumentProcessor.cs

```diff
@@ -24,12 +24,15 @@ public class RestApiDocumentProcessor : DisposableDocumentProcessor
     {
         private const string RestApiDocumentType = "RestApi";
         private const string DocumentTypeKey = "documentType";
-        private static readonly string[] SupportedFileEndings = new string[]
+
+        // To keep backward compatibility, still support and change previous file endings by first mapping sequence
+        private static readonly string[] SupportedFileEndings =
         {
            "_swagger2.json",
```

Comment:
originally, if a file is 'a.b_swagger2.json', it would be change file name to 'a.b' acoording to line 115, will add more comments here to keep backward compatibility

---
In reply to: [73649658](https://github.com/dotnet/docfx/pull/576#discussion_r73649658) [](ancestors = 73649658)